config-controller/pkg/client/client.go

```diff
@@ -295,8 +295,8 @@ func (c *client) DeletePolicy(ctx context.Context, name string) error {
 }
 
 func (c *client) FlushCache(ctx context.Context) error {
-	if time.Since(c.lastUpdated).Seconds() < 10 {
-		// Don't flush the cache more often than every 10s
+	if time.Since(c.lastUpdated).Seconds() < 1 {
```

Comment:
revert before merging? assuming this is not intentional?

Reply:
Having this at 10 seconds delays the integration test by 10 seconds.

The e2e test case we have implemented breaks the cache.  A policy is created in Central and the test immediately turns around and tries to add that policy as a CR.  In an e2e testing environment, it's very likely that the cache will have been updated less than 10s before the test executes, so that one test ends up waiting up to 10s to complete.

This becomes exponentially worse with a higher `-count` value.

Setting the delay to 1s instead of 10 is a compromise in that tests will still be slightly delayed, yet the most egregious abuse of the API is avoided (refreshing once every second is not as bad as refreshing with no delay at all).